externs/shaka/abr_manager.js

```diff
@@ -94,6 +94,22 @@ shaka.extern.AbrManager = class {
    */
   getBandwidthEstimate() {}
 
+  /**
+   * Updates manager playback rate.
+   *
+   * @param {number} rate
+   * @exportDoc
+   */
+  setPlaybackRate(rate) {}
+
+  /**
+   * Return the current playback rate.
+   *
+   * @return {number}
+   * @exportDoc
+   */
+  getPlaybackRate() {}
```

Comment:
This doesn't seem to be necessary, and it implies that AbrManager controls playbackRate.  Instead, AbrManager simply gets notified of it when it changes.

If you were thinking in terms of getter/setter, you could instead follow the naming convention used above in segmentDownloaded, and change setPlaybackRate to something like playbackRateChanged.

Reply:
I have simplified the logic and now there shouldn't be this control by the AbrManager